api/DocumentFragment.json

```
@@ -77,7 +77,7 @@
               "version_added": true
             },
             "safari": {
-              "version_added": false
+              "version_added": true
```

I tracked down the WebKit commit and dated it to Safari 8: https://trac.webkit.org/changeset/162062/webkit

```suggestion
              "version_added": "8"
```

---

Thank you!